azext_edge/edge/providers/orchestration/targets.py

```diff
@@ -150,7 +153,11 @@ def get_ops_enablement_template(
             },
             template_blueprint=M3_ENABLEMENT_TEMPLATE,
         )
-
+        if self.user_trust:
+            # disable cert and trust manager
+            parameters["trustConfig"]["value"]["source"] = "CustomerManaged"
+            # patch enablement template expecting full trust settings for source: CustomerManaged
+            del template.content["definitions"]["_1.CustomerManaged"]["properties"]["settings"]
```

Comment:
Alternatively, what about

`template.content["definitions"]["_1.CustomerManaged"]["properties"]["settings"]["nullable"] = True`
or 
`template.get_type_definition("_1.CustomerManaged")["properties"]["settings"]["nullable"] = True` for that matter